### ISSUE TITLE
Option to remove "paste" field

### ISSUE THREAD

**Issue by cudeso** [MISP/misp-modules#63]
Hi,

An option in the import modules to remove the "paste data" textbox. 
In some cases it doesn't make sense (for example import module that takes an external ID to fetch data but no data to paste).

kr,

koen

**Comment by cudeso** [comment]
Can be closed; fixed by removing the 'paste' from inputSource = []

**Comment by cudeso** [comment]
Doh, removing 'paste' doesn't show the textbox but submit returns "Invalid file upload."

**Comment by iglocska** [comment]
Just to make sure I follow: Currently there are 2 types of input: File upload and paste. Using inputSource you can have one or the other or both - do you mean you'd like a 3rd option?

**Comment by cudeso** [comment]
Maybe that's the best way, an inputSource that only shows the userConfig defined input variables, without paste-field (or file upload field).
Alternative, a flag in userConfig that says "nopaste" (or "nofile") .

It's more a "visual" thing, leaving the paste field blank works when submitting values in userConfig.

**Comment by iglocska** [comment]
Thanks for the heads-up on both issues, I've just pushed a fix that should
allow you to use an empty list for inputSource. This will simply not create
a paste field nor a file upload element and will purely pass the userconfig
data back (with the data field being empty but set).

For the other issue you're correct, the module system was not passing the
server settings related to the module back, I've just added that to what is
being passed.

Let me know if you run into any other issues with it!

Best regards,
Andras
